src/components/Footer/index.jsx

```diff
@@ -12,14 +12,16 @@ import logo from '../../assets/images/africai.png';
 
 library.add(fab, faCoffee);
 const Footer = () => (
-  <footer>
-    <div className="flex_container">
-      <div className="col">
-        <img src={logo} alt="Africa I" className="footer_logo" />
+  <footer className="footer">
+    <div className="footer-container">
+      <div className="footer-subcontainer">
+        <Link to="/" className="brand-logo">
+          <img src={logo} alt="Africa I" className="footer-logo" />
+        </Link> 
       </div>
-      <div className="col">
-        <h4 className="sub_heading">Quick links</h4>
-        <ul className="links__list">
+      <div className="footer-subcontainer">
```

Comment:
@eaprestige  is this `BEM` format? I think it should be like `footer__subcontainer`. BEM uses double underscores for element and double dashes for a modifier.

Reply:
Okay i will fix that right away @eliemugenzi .

Reply:
Good, keep it up.